modules/react/modal/stories/examples/CustomFocus.tsx

```diff
@@ -12,31 +12,31 @@ export const CustomFocus = () => {
     initialFocusRef: ref,
   });
 
-  const handleDelete = () => {
-    console.log('Deleted item');
+  const handleClose = () => {
+    console.log('Close modal');
   };
 
   return (
     <Modal model={model}>
-      <Modal.Target as={DeleteButton}>Delete Item</Modal.Target>
+      <Modal.Target as={PrimaryButton}>Open Modal</Modal.Target>
       <Modal.Overlay>
         <Modal.Card>
           <Modal.CloseIcon aria-label="Close" />
-          <Modal.Heading>Delete Item</Modal.Heading>
-          <Modal.Body>
+          <Modal.Heading>Modal Heading</Modal.Heading>
+          <Modal.Body paddingBottom="zero">
             <Box as="p" marginTop={0} marginBottom="m">
               Enter name to confirm deletion
             </Box>
```

Comment:
I'm curious why did you change heading and button texts? New heading text is not connected to content it has. Maybe text should be abstract text as you had for heading and buttons, such it has in CustomTarget story.

Reply:
Yeah, i was just trying to go off of this https://chakra-ui.com/docs/components/modal/usage#usage which is very generic

Reply:
I can add some lorem ipsum!

Reply:
Also, it seems like @emroller16 did something close to this: https://github.com/Workday/canvas-kit/pull/1621/files

Reply:
Yeah, I followed up with her, she's on PTO, if she wants to commit that change, I'll go ahead and keep the stories as they where and let her make the change!

Reply:
Awesome! 👍